.github/ISSUE_TEMPLATE/release-checklist.md

```diff
@@ -9,7 +9,19 @@ Tagging:
  - [ ] Push that tag to GitHub
 
 Packaging:
- - [ ] Create a PR to bump the Ignition spec file in [Fedora](https://src.fedoraproject.org/rpms/ignition).
+ - [ ] Update the Ignition spec file in [Fedora](https://src.fedoraproject.org/rpms/ignition):
+   - Update the commit hash global variable
+   - Bump the `Version`
+   - Switch the `Release` back to `1.git%{shortcommit}%{?dist}`
+   - Remove any patches obsoleted by the new release
+   - Run `go-mods-to-bundled-provides.py | sort` while inside of the Ignition directory you ran `./tag_release` from & copy output into spec file in `# Main package provides` section
+   - Add any new spec paths to `%gotest` lines
+   - Update change log
+ - [ ] Run `./download_ignition.sh`
+ - [ ] Run `kinit your_fas_account@FEDORAPROJECT.ORG`
+ - [ ] Run `fedpkg upload tarball-name`
+ - [ ] Edit `./sources` to delete the old tarball
```

Comment:
You can combine those two with `fedpkg new-sources tarball-name`.

Reply:
nifty, updated.